portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -76,10 +82,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = getStringParameter(request, TEXT_INPUT_PARAMETER, DEFAULT_STRING);
+    // get User Email
+    if (!userService.isUserLoggedIn()) {
+      return;
```

Comment:
I think we still need to return a response or redirect here, otherwise the user will just get stuck on a blank page? perhaps refactor a bit, e.g.

```java
if (userService.isUserLoggedIn()) {
  // Add comment entity to Datastore
  // ...
}
// Redirect to about me page on post
response.sendRedirect("/index.html");
```